src/utilities/message/messageParser.ts

```diff
@@ -39,7 +45,7 @@ export class MessageParser {
       });
 
       result.response.errors = errors;
-      result.response.message = response.content.errors ? response.content.message : 'Er is een fout opgetreden';
+      result.response.message = response.content.errors ? response.content.message : customMessage;
```

Comment:
moet de check eingelijk niet op customMessage zijn? dus als er customMessage is die tonen, anders de default stuff?

Reply:
funky is how we roll